dev/com.ibm.ws.concurrent_fat_jakarta/test-applications/ConcurrencyTestApp/resources/META-INF/application.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Need copyright header?

Reply:
Thanks for catching this. I added it under https://github.com/OpenLiberty/open-liberty/pull/19240/commits/fff368d5ed72a8e94a9bfdc024d82aa3b25620fc